### PR TITLE
add relevance inspector

### PR DESCRIPTION
**Pull request by dlutzCoveo**

### CODE REVIEW

**Comment by dlutzCoveo** [comment]
Implementing the relevance inspector on v0.13.0 Headless CSB example.

**Comment by dlutzCoveo** [comment]
https://codesandbox.io/s/wizardly-https-03wuy

File: package.json [hunk 1]
```diff
@@ -1,17 +1,18 @@
 {
-  "name": "codeo-headless-demo-080",
+  "name": "coveo-headless-v0130-relevance-inspector",
   "version": "1.0.0",
   "description": "",
   "keywords": [],
   "main": "src/index.tsx",
   "dependencies": {
-    "@coveo/headless": "0.8.0-alpha.9",
```
Comment (jpmarceau) [inline package.json:8]: Weird, I thought this had already been set to 13 🤔

File: src/Components/RelevanceInspectorWindow.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  backgroundColor: "#000"
+};
+
+function RelevanceInspectorWindow(props) {
+  //const { classes } = this.props;
+  const classes = useStyles();
+  const { open, setOpen, json, expandAll } = props;
+
+  const handleClose = () => {
+    console.log("handleClose");
```
Comment (jpmarceau) [inline src/Components/RelevanceInspectorWindow.tsx:35]: This line was probably for debugging purposes and can be removed?

**Review by jpmarceau — APPROVED** [review]
tried it and it works like a charm 👍

**Pull request merged**